core/controllers/payload_validator.py

```diff
@@ -248,20 +244,20 @@ def validate(handler_args, handler_args_schemas, allowed_extra_args):
     'UsernameCheckHandler',
     'ValidateExplorationsHandler',
     'ValueGeneratorHandler',
-    'VoiceArtistManagementHandler'
-    ]
-
-# These handlers do not require any schema validation.
-HANDLER_CLASS_NAMES_WHICH_DO_NOT_REQUIRE_SCHEMAS = [
-    'SessionBeginHandler',
-    'SessionEndHandler',
+    'VoiceArtistManagementHandler',
     'OppiaMLVMHandler',
     'CsrfTokenHandler',
     'Error404Handler',
     'FrontendErrorHandler',
     'WarmupPage',
     'HomePageRedirectPage',
-    'SplashRedirectPage',
+    'SplashRedirectPage'
+    ]
+
+# These handlers do not require any schema validation.
+HANDLER_CLASS_NAMES_WHICH_DO_NOT_REQUIRE_SCHEMAS = [
```

Comment:
@rohitkatlaa I think we should not remove this entirely because in the backend test we iterate over all the routes, whether the handler is a child class of BaseHandler or not.
So if we remove this list -- the backend tests will fail for those which are not the child class of BaseHandler.
For now I have seperated the handlers into two lists -- 
1. The one which are child class of BaseHandler.
2. The one which are not the child class of BaseHandler.

Looking forward for your suggestions.

Reply:
Please rename the constant and update the comment appropriately.

Reply:
I have updated comment and now comment and list name matches with each other. 
Please suggest a name which is more appropriate in this context.

Reply:
Actually there is another handler that will be added according to this [discussion](https://github.com/oppia/oppia/pull/13533#discussion_r684295034) so could you update the comment appropriately. Thanks.

Reply:
As per offline discussion, this comment is reverted to initial comment.
Please take a look.

Reply:
Thank you Nikhil!